test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1997,6 +2029,42 @@ static public void testDefaultValueInLineArray() throws Throwable {
 		}
 	}
 
+	/**
+	 * Create multi dimensional array with Line Class without initialization.
+	 * Check the fields of each element in arrays. No field should be NULL.
+	 */
+	@Test(priority=4)
+	static public void testDefaultValueInLineInstanceMultiANewArray() throws Throwable {
```

Comment:
for this test and the ones below, you can first do a loop write out each element with `defaultLinePositions1` (look at `checkFieldAccessMHOfAssortedType`) and then do another loop to compare that it is correct with `checkFieldAccessMHOfAssortedType`. Look at `testGCFlattenedValueArrayWithSingleAlignment` for an example.